prow/config.yaml

```diff
@@ -9948,6 +9948,19 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180418-79a97ca02-master
 
+- interval: 4h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gci-gce-kube-dns
```

Comment:
These jobs will need to live in somewhere like test-infra/config/jobs/kubernetes/sig-foo/foo-config.yaml now.

Reply:
Probably use https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-network/sig-network-misc.yaml.